ui/src/i18n/pl.json

```diff
@@ -50,28 +50,28 @@
   "accessories.message_settings_link": "Możesz uzyskać dostęp do ustawień Homebridge w dowolnym momencie z menu głównego lub klikając cog",
   "accessories.title_accessories": "Akcesoria",
   "accessories.title_accessory_control_disabled": "Sterowanie akcesoriami jest wyłączone",
-  "backup.button_download_backup_archive": "Utwórz backup",
-  "backup.button_restore_backup": "Przywróć backup",
-  "backup.label_archive_name": "Nazwa backupu",
-  "backup.label_backup_size": "Rozmiar backupu",
+  "backup.button_download_backup_archive": "Utwórz kopię zapasową",
```

Comment:
"Backup" is definitely more popular than "kopia zapasowa".

Reply:
I think „Backup” it’s not a Polish name. I think is English, right?

Reply:
You say in Poland "hot-dog" or "gorący pies"? :)